From: Benoît Knecht <benoit.knecht@fsfe.org>
Subject: Use /var/cache/minidlna as the default db_dir

---
 minidlna.conf | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/minidlna.conf b/minidlna.conf
index 42cb72a..e28cbdb 100644
--- a/minidlna.conf
+++ b/minidlna.conf
@@ -24,7 +24,7 @@
 media_dir=/var/lib/minidlna
 
 # Path to the directory that should hold the database and album art cache.
-#db_dir=/var/lib/minidlna
+#db_dir=/var/cache/minidlna
 
 # Path to the directory that should hold the log file.
 #log_dir=/var/log
