From: Benoît Knecht <benoit.knecht@fsfe.org>
Subject: Make sure the database is closed after scanning

---
 minidlna.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/minidlna.c b/minidlna.c
index b4a3770..4c77133 100644
--- a/minidlna.c
+++ b/minidlna.c
@@ -374,6 +374,7 @@ rescan:
 		open_db(&db);
 		if (*scanner_pid == 0) /* child (scanner) process */
 		{
+			open_db(&db);
 			start_scanner();
 			sqlite3_close(db);
 			log_close();
@@ -386,6 +387,7 @@ rescan:
 		}
 #else
 		start_scanner();
+		sqlite3_close(db);
 #endif
 	}
 }
